data.json

```diff
@@ -1,46 +1,46 @@
-        "land": ["Albanien", "Armenien", "Andorra", "Algerien"],
+        "stadt": ["Aalen", "Aachen", "Amberg", "Athen (Griechenland)", "Aschaffenburg", "Augsburg", "Amsterdam"],
+        "land": ["Albanien", "Armenien", "Andorra", "Algerien", "Australien", "Argentinien"],
         "fluss": ["Aller (Deutschland)", "Amazonas (Brasilien)", "Alz (Deutschland)", "Ätran (Schweden)", "Aa (Deutschland)"],
         "name": ["Alexander", "Alexandra", "Anette", "Amanda", "Anton"],
         "beruf": ["Atomphysiker", "Anlagenbauer", "Autor", "Astronaut"],
-        "tier": ["Aul", "Affe", "Ara", "Ameise", "Amaisenbär"],
+        "tier": ["Aul", "Affe", "Ara", "Ameise", "Amaisenbär", "Aal"],
         "marke": ["Adidas", "Abus", "Alpina", "Asics"],
         "pflanze": ["Agave", "Apfel", "Avocado", "Ananas", "Ahorn", "Alge"]
     },
     "b": {
-        "stadt": ["Bamberg", "Bonn", "Böblingen", "Boston", "Birmingham"],
-        "land": ["Belgien", "Brasilien", "Bolivien", "Bermudas", "Bahrain"],
+        "stadt": ["Bamberg", "Bonn", "Böblingen", "Boston", "Birmingham", "Bad Kreuznach", "Bayreuth", "Berlin"],
+        "land": ["Belgien", "Brasilien", "Bolivien", "Bermudas", "Bahrain", "Bosnien", "Belize"],
         "fluss": ["Bigge (Deutschland)", "Brahmaputra (u.a. Indien)", "Belaja (Russland)", "Blies (Frankreich und Deutschland)"],
-        "name": ["Benjamin", "Benny", "Bernd", "Barbara", "Beate"],
-        "beruf": ["Busfahrer", "Beamter", "Bundesminister"],
-        "tier": ["Bär", "Biene", "Beluga", "Biber", "Bea", "Bison", "Blaumeise"],
-        "marke": ["Basil", "Barbour", "Bench", "Billa Bong", "Blomus", "Bosch", "Braum"],
-        "pflanze": ["Berlauch", "Bartlilie"]
+        "name": ["Benjamin", "Benny", "Bernd", "Barbara", "Beate", "Boris", "Birgit"],
+        "beruf": ["Busfahrer", "Beamter", "Bundesminister", "Bäcker" , "Bankkaufmann", "Bauzeichner", "Bestatter"],
+        "tier": ["Bär", "Biene", "Beluga", "Biber", "Bea", "Bison", "Blaumeise", "Blauwal", "Blindschleiche"],
+        "marke": ["Basil", "Barbour", "Bench", "Billa Bong", "Blomus", "Bosch", "Braum", "BMW", "Bogner"],
+        "pflanze": ["Bärlauch", "Bartlilie", "Begonie"]
     },
     "c": {
         "stadt": ["Chemnitz (Deutschland)", "Cuxhaven (Deutschland)", "Castrop-Rauxel (Deutschland)", "Charleston (USA)", "Calvi (Frankreich)"],
         "land": ["China", "Costa Rica", "Chile", "Cookinseln"],
-        "fluss": ["Chemnitz (Deutschland)", "Colorado (USA)", "Charente (Frankreich)", "Churchill River (Kanada)"],
+        "fluss": ["Chemnitz (Deutschland)", "Colorado (USA)", "Charente (Frankreich)", "Churchill River (Kanada)", "Cooper Creek (Australien)"],
         "name": ["Christian", "Claus", "Christoph", "Conni", "Clara"],
-        "beruf": ["Chemiker", "Chorleiter"],
-        "tier": ["Chameleon", "Chinchilla", "Chiuahua"],
+        "beruf": ["Chemiker", "Chorleiter", "Chauffeur", "Chirurg"],
+        "tier": ["Chameleon", "Chinchilla", "Chiuahua", ""],
```

Comment:
Can you please remove this empty string?

Reply:
@xRealNeon Done, see new pull Request.